File: Visual VB.NET/Forms Applications/Demo/SMPPServerMessage.vb
Imports AxSms

Public Class SMPPServerMessage
    Private strSystemID As String
    Private objMessageToSend As AxSms.Message

    Public Property SystemID()
        Get
            Return strSystemID
        End Get
        Set(ByVal value)
            strSystemID = value
        End Set
    End Property

    Public Property MessageToSend()
        Get
            Return objMessageToSend
        End Get
        Set(ByVal value)
            objMessageToSend = value
        End Set
    End Property

    Public Sub New(ByVal strSystemID As String, ByVal objMessageToSend As AxSms.Message)
        SystemID = strSystemID
        MessageToSend = objMessageToSend
    End Sub

End Class

File: Visual CSharp.NET/Forms Applications/Demo/SMPPServerMessage.cs
using System;
using System.Collections.Generic;
using System.Text;
using AxSms;

namespace Demo
{
    class SMPPServerMessage
    {
        private string strSystemID;
        private AxSms.Message objMessageToSend;

        public string SystemID
        {
            get { return strSystemID; }
            set { strSystemID = value; }
        }        

        public AxSms.Message MessageToSend
        {
            get { return objMessageToSend; }
            set { objMessageToSend = value; }
        }

        public SMPPServerMessage(string strSystemID, AxSms.Message objMessageToSend)
        {
            this.strSystemID = strSystemID;
            this.objMessageToSend = objMessageToSend;
        }

    }
}
